apps/docs/_data/components.json

```
@@ -280,7 +280,7 @@
 		]
 	},
 	{
-		"title": "option",
+		"title": "listbox-option",
```

How does it relate to this PR?

---

it doesn't. caught red handed
```suggestion
		"title": "option",
```